### PR TITLE
Ports LETPASSCLICKS from Daedalus Dock

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
A deal with @Fikou 
- https://github.com/DaedalusDock/daedalusdock/pull/106

In layman's terms, you can grab this wrench now.
205561405-902f1795-0c63-4dab-8435-d8902f9c4499

Adds a new pass_flags_self value for click transparency, allowing objects to be dense without blocking click attempts.
## Changelog
:cl:
add: Lockers, crates, and machines no longer block click attempts in adjacency checks. Basically, you can reach tables cornered between lockers/machines.
/:cl:

### CODE REVIEW

File: code/__DEFINES/flags.dm [hunk 1]
```diff
@@ -155,6 +155,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define PASSDOORS (1<<10)
 #define PASSVEHICLE (1<<11)
 #define PASSITEM (1<<12)
+/// Do not intercept click attempts during Adjacent() checks. See [turf/proc/ClickCross]. **ONLY MEANINGFUL ON pass_flags_self!**
+#define LETPASSCLICKS (1<<13)
```
Comment (Mothblocks) [inline code/__DEFINES/flags.dm:159]: Should be LET_PASS_CLICKS, I know the rest are like this but they should not be

**Review by Mothblocks — APPROVED** [review]

**Review by spookydonut — APPROVED** [review]

**Comment by ZephyrTFA** [comment]
Why approve it twice, just merge it

**Comment by Mothblocks** [comment]
@ZephyrTFA they didnt do my change :(
